docusaurus.config.js

```diff
@@ -69,12 +69,31 @@ const config = {
     ['docusaurus-plugin-sass',{}],
     // ------------DOCUSAURUS MULTI-INSTANCE PLUGIN--------------
     [
-      '@docusaurus/plugin-content-docs',
-      {
+      '@docusaurus/plugin-content-docs', {
         id: 'docs-kits',
         path: 'docs-kits',
         routeBasePath: 'docs-kits',
         sidebarPath: require.resolve('./sidebarsDocsKits.js'),
+        onlyIncludeVersions: ['25.06', '25.03', '24.12', 'current'],
+        lastVersion: 'current',
+        versions: {
+          'current': {
+            label: 'Next',
+            banner: 'unreleased'
+          },
+          '25.06': {
+              label: '25.06',
+              banner: 'none'
+          },
+          '25.03': {
+            label: '25.03',
+            banner: 'none'
+          },
+          '24.12': {
```

Comment:
since we still have the version folder for 24.08 ... shouldnt it be also in this list?